.github/workflows/docs.yml

```diff
@@ -0,0 +1,38 @@
+name: docs
+on:
+  push:
+    branches: [main]
+    paths: ['src/**/*.rs', 'Cargo.toml', '.github/workflows/docs.yml']
```

Comment:
Why is there `Cargo.toml` too?

Reply:
I am not sure how `cargo doc` works in details.
I think the `workspace.members` key of the `Cargo.toml` will influence in any case the crates included in the docs...
hence triggering the docs workflow if this file is modified.

Reply:
Let's keep it then!